chain-signatures/node/src/storage/presignature_storage.rs

```diff
@@ -7,18 +8,36 @@ use crate::storage::error::{StoreError, StoreResult};
 
 // Can be used to "clear" redis storage in case of a breaking change
 const PRESIGNATURE_STORAGE_VERSION: &str = "v2";
+const USED_EXPIRE_TIME: Duration = Duration::hours(24);
 
 pub fn init(pool: &Pool, node_account_id: &AccountId) -> PresignatureStorage {
```

Comment:
I don’t know why we kept using this pattern but this is normally reserved for functions that return more than one value such as a tuple. It’s better in the future for us to have a `PresignatureStorage::new` instead but this isn’t blocking anything

Reply:
I'm trying to keep everything consistent. Ok, I will take a look.